sdk/canton/community/base/src/main/protobuf/com/digitalasset/canton/protocol/v3/participant_transaction.proto

```diff
@@ -32,7 +32,7 @@ message ActionDescription {
   oneof description {
     v0.ActionDescription.CreateActionDescription create = 1;
     ExerciseActionDescription exercise = 2;
-    v1.ActionDescription.FetchActionDescription fetch = 3;
+    FetchActionDescription fetch = 3;
```

Comment:
Looking at `object ActionDescription.supportedProtoVersions`, I see that proto version 3 is tied to ProtocolVersion 6. Since PV6 is already released, you should not change this anymore.

So you essentially need to do all this in proto version 4 and tie it to PV7.

Reply:
Have done this now.